src/gradata/enhancements/behavioral_extractor.py

```
@@ -0,0 +1,493 @@
+
+def _is_actionable(instruction: str) -> bool:
+    if not instruction or len(instruction) < 5:
+        return False
+    first_word = instruction.split()[0].lower().removesuffix("'t")
+    if first_word in _IMPERATIVE_STARTERS:
+        return True
+    # Accept instructions from PREFIX_INSTRUCTION archetype (explicit user rules)
+    # and any instruction that looks imperative (capitalized verb form)
+    if instruction[0].isupper() and len(instruction.split()) >= 3:
+        return True
+    return False
```

Ruff (0.15.9)

[warning] Return the condition directly

Inline condition

(SIM103)